.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+# Description
+<!--- 
+- a short summary of the pull request (PR)
```

Comment:
do people really want to write a description **and** changes? 🤔

Reply:
Good point. I removed the description section and flattened the ticket no, open questions and uncovered ACs into separate sections in cfcf40000b0aee5f24c9e4ccaf877f3a85e2209e